web/viewer.css

```diff
@@ -1428,10 +1429,33 @@ html[dir='rtl'] .outlineItemToggler::before {
 #passwordOverlay > .dialog {
   text-align: center;
 }
+
 #passwordOverlay .toolbarField {
   width: 200px;
 }
 
+@media print {
+  #mozPrintCallback-shim {
+    display: none;
+    page-break-after: always;
+    page-break-inside: avoid;
```

Comment:
I'd expect that placing the `page-break-...` rules unconditionally in `viewer.css` will actually cause issues in e.g. FIrefox, since they were purposely moved in PR #5785.